python_modules/dagster/dagster/core/execution/stats.py

```diff
@@ -163,6 +165,8 @@ def build_run_step_stats_from_events(
             step_attempts.append(
                 RunStepMarker(start_time=attempt_start, end_time=step_stats["end_time"])
             )
+        else:
+            by_step_key[step_key]["status"] = StepEventStatus.IN_PROGRESS
```

Comment:
sorry, this is the line that I think will affect the `verify_step` call.  Put another way, prior to this change, the only steps that returned from this call were completed steps.  Now we're changing the logic to have it include in progress steps.  We need to make sure that all callers of this can handle this.